schema.sql

```diff
@@ -1,6 +1,25 @@
 /* Database schema to keep the structure of entire database. */
 
-ALTER TABLE animals ADD COLUMN species TEXT;
+
+/* Create a table named species */
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(255),
+  age INTEGER
+);
+
+/* create a table named species */
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255)
+);
+
+/* modify animals table */
+ALTER TABLE animals
+ADD COLUMN species_id INTEGER,
+ADD COLUMN owner_id INTEGER,
+DROP COLUMN species;
```

Comment:
- [ ] Kindly include the code that creates the `animals` table. If someone is cloning your repository newly, they would need to see the code that creates the `animals` table. Kindly include it 👍